<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertFeatureDlg</class>
 <widget class="QWidget" name="InsertFeatureDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>483</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Available Feature Commands:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="featureCmdList"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="featureCmdHelp_layout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Feature description</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="featureNameLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Feature name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="featureName">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>newFeature</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Statement type</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="isIntermediateStep">
          <property name="text">
           <string>Intermediate modelling step</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="isFinalComponent">
          <property name="text">
           <string>Final assembly component</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="onlyFeatureCommand">
          <property name="text">
           <string>Insert feature command only</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
